File: stepik-union/src/main/java/org/stepik/core/courseFormat/stepHelpers/DatasetQuizHelper.java
package org.stepik.core.courseFormat.stepHelpers;

import com.intellij.openapi.project.Project;
import org.jetbrains.annotations.NotNull;
import org.stepik.core.courseFormat.StepNode;

/**
 * @author meanmail
 */
public class DatasetQuizHelper extends QuizHelper {
    public DatasetQuizHelper(@NotNull Project project, @NotNull StepNode stepNode) {
        super(project, stepNode);
    }

    @NotNull
    public String getData() {
        initStepOptions();
        return reply.getText();
    }
}

File: stepik-union/src/main/java/org/stepik/core/courseFormat/stepHelpers/DatasetQuizHelper.kt
package org.stepik.core.courseFormat.stepHelpers

import com.intellij.openapi.project.Project
import org.stepik.core.courseFormat.StepNode


class DatasetQuizHelper(project: Project, stepNode: StepNode) : QuizHelper(project, stepNode) {

    val data: String
        get() {
            initStepOptions()
            return reply.file
        }

    override val isAutoCreateAttempt: Boolean
        get() = false
}
